tasks/configure.yml

```diff
@@ -19,9 +20,11 @@
     src: cloudwatch.yml.j2
   notify:
     - restart prometheus cloudwatch exporter
+  when: ansible_user_id != "vagrant" and ansible_user_id != "travis"
 
 - name: Prometheus CloudWatch Exporter service start
   become: yes
   service:
     name: prometheus-cloudwatch-exporter
     state: started
+  when: ansible_user_id != "vagrant" and ansible_user_id != "travis"
```

Comment:
I think it's always more readable like this (line 30, 23, and 11):

```yaml
when:
  - ansible_user_id != "vagrant"
  - ansible_user_id != "travis"
```

Reply:
I agree it looks more readable, however whenever I've tried it that way it doesn't work.